src/main/java/com/ionosenterprise/sdk/IPBlock.java

```diff
@@ -94,4 +98,23 @@ public IPBlocks getAllIPBlocks() throws RestClientException, IOException {
    public void deleteIPBlock(String ipBlockId) throws RestClientException, IOException {
       client.delete(getUrlBase().concat(resource).concat("/").concat(ipBlockId), 202);
    }
+
+   /**
+    * Update the name of an IP block.
+    *
+    * @param ipBlock object has the following properties:
+    * <br>
+    * <br>
+    * name = A descriptive name for the IP block
+    * <br>
+    * <br>
+    * @return IPBlock object with properties and metadata.
+    */
+   public com.ionosenterprise.rest.domain.IPBlock updateIPBlock(String ipBlockId,
+                                                                com.ionosenterprise.rest.domain.IPBlock.Properties ipBlock)
```

Comment:
Wouldn't it be better to rename `ipBlock` as `properties` or `ipBlockProperties`?

Reply:
Fixed.